src/test/java/org/mockito/configuration/MockitoConfiguration.java

```diff
@@ -0,0 +1,12 @@
+package org.mockito.configuration;
+
+/**
+ * This is to clean runners between tests.
+ */
+public class MockitoConfiguration extends DefaultMockitoConfiguration {
```

Comment:
@uKL earlier in the PR is saw you used some commandline arguments? What was that? 
Having problems using this when running lots of tests, looks like he doesnt use the configuration provided.